src/test/java/unit/BqDatasetReferenced.java

```diff
@@ -309,60 +309,63 @@ void updateIndividualProperties() throws IOException {
     // update just the name
     // `terra resources update bq-dataset --name=$name --new-name=$newName`
     String newName = "updateIndividualProperties_NEW";
-    UFBqDataset updateDataset =
+    UFBqDataset updatedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class,
             "resource",
             "update",
             "bq-dataset",
             "--name=" + name,
             "--new-name=" + newName);
-    assertEquals(newName, updateDataset.name);
-    assertEquals(description, updateDataset.description);
+    assertEquals(newName, updatedDataset.name);
+    assertEquals(description, updatedDataset.description);
 
     // `terra resources describe --name=$newName`
-    UFBqDataset describeDataset =
+    UFBqDataset describedDataset =
         TestCommand.runAndParseCommandExpectSuccess(
             UFBqDataset.class, "resource", "describe", "--name=" + newName);
-    assertEquals(description, describeDataset.description);
+    assertEquals(description, describedDataset.description);
```

Comment:
For description:

line 327: Assert old description
line 333: Update new description
line 349: Assert new description

You only have:

line 334: Update cloning instructinos
line 350: Assert new cloning instructions

You're missing:

line 328: assert old cloning instructions

For all we know, update could be broken and cloning instructions could have been set to NOTHING originally. line 328 makes us certain that updating works.

Please do this for all tests.